cli-tests/scripts/dts-import.js

```diff
@@ -0,0 +1,50 @@
+import { resolve } from 'path';
+import { CUSTOM_TRANSFER_TOKEN_ACCESS_KEY } from '../constants';
+
+const {
+  file: {
+    providers: { createLocalFileSourceProvider },
+  },
+  strapi: {
+    providers: { createRemoteStrapiDestinationProvider },
+  },
+  engine: { createTransferEngine },
+} = require('@strapi/data-transfer');
+
+/**
+ * Reset the DB and import data from a DTS dataset
```

Comment:
Could you add instructions as you did with the export?

Reply:
Done